### DISCUSSION TITLE
[Esame 13 Settembre 2021] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#262] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-14 112640](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/38f7474d-01b5-4607-908e-e8867416b201)

**Comment by Elia-Belli** [comment]
**(1)** Otteniamo $1$ per la prima volta alla quinta estrazione se non lo estraiamo nelle precedenti e lo troviamo ora, inoltre ad ogni estrazione la probabilità varia:

$$\mathbb{P}(A)=\frac{48}{52}\cdot \frac{47}{51}\cdot \frac{46}{50}\cdot \frac{45}{49}\cdot \frac{4}{48}=4\cdot \frac{48!}{44!}\cdot \frac{47!}{52!}\simeq 0.06$$

**(2)** Utilizziamo la notazione $XXXXX$ per indicare l'ordine delle estrazioni:

$$\mathbb{P}(2222|A)=\frac{\mathbb{P}((2222) \cap A)}{\mathbb{P}(A)}=\frac{\mathbb{P}(22225)}{\mathbb{P}(A)}=\frac{\frac{4}{52}\cdot \frac{3}{51}\cdot \frac{2}{50} \cdot \frac{1}{49}\cdot \frac{4}{48}}{\frac{48}{52}\cdot \frac{47}{51}\cdot \frac{46}{50}\cdot \frac{45}{49}\cdot \frac{4}{48}}=24\cdot \frac{44!}{48!}\simeq 5.13\cdot 10^{-6}$$

**(3)** Possiamo generalizzare partendo dal punto **(1)**, indichiamo con $X$ la v.a che indica l'estrazione in cui si è trovato il primo asso:

$$\mathbb{P}(X=k)=4\cdot \frac{48!}{(48-k+1)!}\cdot \frac{(52-k)!}{52!}\ \ \ \ \ \ \ k=1,...,49$$

> Possiamo anche utilizzare la probabilità composta

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/ff0f144f-091b-4880-b53c-c6b3c02a64d1)
